docs/adr/0045-testing-updates-and-recommendations.md

```diff
@@ -0,0 +1,233 @@
+# Improving Testing Standards and Experience in the Monorepo
+
+- Status: Proposed
+- Deciders: Wil Clouser, Dan Schomburg, Valerie Pomerleau, Barry Chen, Amri Toufali, Vijay Budhram
+
+## Context and Problem Statement
+
+The current testing ecosystem in the FxA is fragmented and inconsistent, leading to unnecessary cognitive overhead, inefficiencies in test execution, and friction in developer experience. Projects across the mono-repo use a mix of testing frameworks—primarily Mocha for legacy code and Jest for newer sub-projects. This inconsistency leads to increased context-switching, duplicated tooling knowledge, and issues when configuring tools like Nx which rely on consistent script naming conventions.
+
+Additionally, while valuable, the functional test suite can consume significant CI resources and any effort to trim it back can help. A number of these tests could be more efficiently executed at the unit or integration test level. And, a [recent discovery][invalid test thread] found that at least one set of tests were running as false positives, not actually testing what we thought they were.
+
+Similarly, the [test result dashboard][test metrics dashboard] that provides insight into metrics like run times and skip rates is partially broken and has not been accurately reporting since late February or early March on a few metrics. This reduces visibility into test health and performance, and provides a false positive on health.
```

Comment:
Maybe instead of "February or early March" we could mention how long something's been broken.  Or include the year for those months?